LEGO1/lego/legoomni/src/common/mxtransitionmanager.cpp

```diff
@@ -105,7 +105,11 @@ MxResult MxTransitionManager::StartTransition(
 			backgroundAudioManager->Stop();
 		}
 
+#ifdef __3DS__
```

Comment:
Let's try to avoid coding platform specific exceptions into LEGO1 module. This in particular seems suited to be a configurable .ini option, it's been part of Rebuilder for instance. If other transitions aren't workable for 3DS we can ignore the ini setting and hardcore it in ISLE module (similar to how I did it with Emscripten)

Reply:
Reading back the frame buffer and doing CPU graphics, then encoding it as a texture is a really big load on 3DS it's likely going to be slower to do a frame then the normal duration of the transition. That and we don't know how to decode the graphics format used by the frame buffer.

If we rewrite the transitions to be less CPU heavy wipeDown and windows could maybe be viable on 3DS but as things stands noAnimation animation is the only one that will work on this port.

Can we do the ini option as a separate thing from 3DS it doesn't seem very port related.

Reply:
I can start working on rewriting a few of the other CPU based graphics effects to be more GPU friendly, but implementing ini options isn't something that excites me.

Reply:
I understand that transitions aren't viable on 3DS. What I'd like to avoid is setting a precedent of platform specific ifdefs in LEGO1.

Reply:
> but implementing ini options isn't something that excites me.

Happy to do it, takes 2 mins

Reply:
How do you propose handeling `E:` ?

Reply:
I'm not sure how it should be handled since I'm not familiar with this port, but I'd suggest going along with this pattern as used by Emscripten 

https://github.com/isledecomp/isle-portable/blob/master/ISLE/isleapp.cpp#L833

Reply:
Yeah I would say that's the only blocker left.